.moban.dt/url_builder.jj2

```diff
@@ -0,0 +1,61 @@
+{% macro api_doc(doc) -%}
+  api.coala.io/en/latest/{{ doc }}.html
+{%- endmacro %}
+
+{% macro user_doc(doc) -%}
+  docs.coala.io/en/latest/{{ doc }}.html
+{%- endmacro %}
+
+{% macro github_file(org, repo, file, branch="master") -%}
+  github.com/{{ org }}/{{ repo }}/blob/{{ branch }}/{{ file }}
+{%- endmacro %}
+
+{% macro cEP(n) -%}
+  {{ github_file("coala", "cEPs", "cEP-%04d.md" % n) }}
+{%- endmacro %}
+
+{% macro gform(id) -%}
+  docs.google.com/forms/d/e/{{ id }}/viewform
+{%- endmacro %}
+
+{% macro gpresentation(id) -%}
+  docs.google.com/presentation/d/{{ id }}/edit
+{%- endmacro %}
+
+{% macro github_issues(q) -%}
+  github.com/issues?q={{ q | urlencode }}
+{%- endmacro %}
+
+{% macro github_pulls(q) -%}
+  github.com/pulls?q={{ q | urlencode }}
+{%- endmacro %}
+
+{% macro github_wiki(org, repo, page) -%}
+  github.com/{{ org }}/{{ repo }}/wiki/{{ page }}
+{%- endmacro %}
+
+{% macro wiki(page) -%}
+  {{ github_wiki("coala", "repo", page) }}
+{%- endmacro %}
+
+{% macro get_url(sl) %}
+  {% if sl.api_doc is defined -%}
```

Comment:
Looks like there's a bug with whatever linter we use for jinja2, this is a valid syntax since we don't want it to add the whitespaces after this block.